src/Centreon/Domain/Acknowledgement/Acknowledgement.php

```diff
@@ -104,18 +111,18 @@ class Acknowledgement
     private $isSticky;
 
     /**
-     * @Serializer\Groups({"ack_service"})
+     * @Serializer\Groups({"ack_main"})*
```

Comment:
why use snake_case ?

Reply:
Honnestly don't know exctly why. But may be related to the yaml convention